postgres/manage.sls

```diff
@@ -15,12 +17,16 @@ include:
 
 {%- endif %}
 
+{%- if needs_client_binaries or postgres.manage_force_reload_modules %}
```

Comment:
I was getting tired of `postgres-reload-modules` running even when nothing had changed. I hid the state behind a conditional that checks if any new client binaries were installed, or if `manage_force_reload_modules` was set to `True`.

*NOTE*: `manage_force_reload_modules` defaults to `True` to preserve the existing behavior.

Reply:
Maybe`mange_force_reload_modules` should default to False to fix  https://github.com/saltstack-formulas/postgres-formula/issues/214  @vunty what do you think?

Reply:
I'm totally fine with setting `mange_force_reload_modules` to `False` by default.